### PR TITLE
Configure sudoers for cap-tool in installer

### PR DESCRIPTION
**Pull request by rfree-d**

### CODE REVIEW

File: src-misc/locale/galaxy42_installer/messages.pot [hunk 1]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2017-02-17 02:32+0000\n"
+"POT-Creation-Date: 2017-07-17 14:31+0000\n"
 "PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
```
Comment (robertoleksy) [inline src-misc/locale/galaxy42_installer/messages.pot]: bad Language-Team

File: src-misc/locale/galaxy42_installer/pl.po [hunk 2]
```diff
@@ -380,3 +380,41 @@ msgstr ""
 #, sh-format
 msgid "L_language"
 msgstr "Language=(PL)"
+
+msgid "L_install_option_sudo_cap_users_title"
+msgstr "Sudo dla cap-tool: wybór użytkowników/grup"
+
+msgid "L_install_option_sudo_cap_users"
+msgstr ""
+"Wybierz użytkowników i/lub grupy które będą mogłyużywać narzędzia "
```
Comment (robertoleksy) [inline src-misc/locale/galaxy42_installer/pl.po]: typo mogłyużywać

File: src-misc/locale/galaxy42_installer/pl.po [hunk 3]
```diff
@@ -380,3 +380,41 @@ msgstr ""
...
+"cap-tool aby dać swoim programom prawa CAPtakie jak CAP_NET_ADMIN (zmiana "
```
Comment (robertoleksy) [inline src-misc/locale/galaxy42_installer/pl.po]: typo CAPtakie

File: src-misc/locale/galaxy42_installer/pl.po [hunk 4]
```diff
@@ -380,3 +380,41 @@ msgstr ""
...
+"tak:\n"
+"%sudo %admin %developers alice"
+
+msgid "L_warning_sudo_cap_install_failed_title $packages_str"
+msgstr "Sudo dla cap-tool: błąd"
+
+msgid "L_warning_sudo_cap_install_failed"
+msgstr ""
+"Sudo dla cap-tool: wystąpił błąd automatycznej konfiguracji sudo "
+"(sudoers).Możesz skonfigurować to ręcznie przez komendę visudo zamiast "
```
Comment (robertoleksy) [inline src-misc/locale/galaxy42_installer/pl.po]: missing space after dot

File: src-misc/locale/galaxy42_installer/pl.po [hunk 5]
```diff
@@ -380,3 +380,41 @@ msgstr ""
...
+"tego."
+
+msgid "L_install_option_sudo_cap_allow_title"
+msgstr "Sudo dla cap-tool: zgadzasz się?"
+
+msgid "L_install_option_sudo_cap_allow"
+msgstr ""
+"Proponujemy zainstalować i skonfigurować narzędzie cap-tool,\n"
+"tak aby zwykły użytkowników (bez root) mógł skompilować nasz program i "
```
Comment (robertoleksy) [inline src-misc/locale/galaxy42_installer/pl.po]: single char should be in new line

File: src-misc/locale/galaxy42_installer/en.po [hunk 6]
```diff
@@ -362,3 +362,40 @@ msgstr ""
...
+msgid "L_install_option_sudo_cap_allow_title"
+msgstr "Sudo for cap-tool: do you agree?"
+
+msgid "L_install_option_sudo_cap_allow"
+msgstr ""
+"We suggest to install and configure tool cap-tool,\n"
+"so that regular user (without root) can build our program and give it "
+"special rights (like CAP_NET_ADMIN) so that without need for using root (nor "
+"sudo) he can run this program with rights to modify networking.\n"
+"Users that you will sellect in a moment, will be allowed (indirectly by some "
```
Comment (mikurys) [inline src-misc/locale/galaxy42_installer/en.po]: typo sellect

File: src-misc/locale/galaxy42_installer/en.po [hunk 7]
```diff
@@ -362,3 +362,40 @@ msgstr ""
...
+"program they own) to gain partiall admin privileges.\n"
```
Comment (mikurys) [inline src-misc/locale/galaxy42_installer/en.po]: typo partiall

File: src-misc/locale/galaxy42_installer/en.po [hunk 8]
```diff
@@ -362,3 +362,40 @@ msgstr ""
...
 msgid "L_language"
 msgstr "Language=(EN)"
+
+msgid "L_install_option_sudo_cap_users_title"
+msgstr "Sudo for cap-tool: user/group selection"
+
+msgid "L_install_option_sudo_cap_users"
+msgstr ""
+"Select the users and/or groups that will be allowed to use our tool cap-tool "
+"to give own programs privilages CAP such as CAP_NET_ADMIN (changes to "
```
Comment (mikurys) [inline src-misc/locale/galaxy42_installer/en.po]: typo privilages

File: src-misc/locale/galaxy42_installer/pl.po [hunk 9]
```diff
@@ -380,3 +380,41 @@ msgstr ""
...
+"nadać mu specjalne prawa (jak CAP_NET_ADMIN) aby bez użycia root (ani "
+"sudo) mógł on uruchomić ten program z prawami modyfikacji sieci.\n"
+"Użytkownicy których wybierzesz za chwilę, będą mogli (pośrednio przez "
+"jakiś swój program) mieć częsciowe prawa administra.\n"
```
Comment (mikurys) [inline src-misc/locale/galaxy42_installer/pl.po]: typo "częsciowe" and "administra"

File: src-misc/locale/galaxy42_installer/pl.po (hunk 2 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/pl.po]: :ok:

File: src-misc/locale/galaxy42_installer/messages.pot (hunk 1 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/messages.pot]: :ok:

File: src-misc/locale/galaxy42_installer/pl.po (hunk 3 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/pl.po]: :ok:

File: src-misc/locale/galaxy42_installer/pl.po (hunk 5 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/pl.po]: "i" \n ?

The end of lines are controlled by text output anyway (or by hard \n).

Not needed

File: src-misc/locale/galaxy42_installer/en.po (hunk 8 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/en.po]: :ok:

File: src-misc/locale/galaxy42_installer/en.po [hunk 10]
```diff
@@ -362,3 +362,40 @@ msgstr ""
...
+"network settings).\n"
+"Usually this is given to trusted users, who will compile our program here.\n"
+"Provide space-separated names of users,\n"
+"also you can give group name by adding character % at beginingfor example "
```
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/en.po]: :ok:

File: src-misc/locale/galaxy42_installer/en.po (hunk 6 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/en.po]: :ok:

File: src-misc/locale/galaxy42_installer/en.po (hunk 7 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/en.po]: :ok:

File: src-misc/locale/galaxy42_installer/pl.po (hunk 9 above)
Comment (rfree-d) [inline src-misc/locale/galaxy42_installer/pl.po]: :ok:

**Comment by rfree-d** [comment]
@robertoleksy works for you on your machine?

**Pull request merged**